plugin/vscode/extension.js

```diff
@@ -358,18 +369,19 @@ class DocumentLinterCollection {
   dispose() {
     logAsyncErrors(async () => {
       await this.disposeAsync();
+      this._performanceWriter.writeToFile();
     });
   }
 }
 exports.DocumentLinterCollection = DocumentLinterCollection;
 
 let toDispose = [];
 
-async function activateAsync() {
+async function activateAsync(ExtensionContext) {
```

Comment:
I think tests need to provide an ExtensionContext in order for this code to work now. Should be easy to whip up a temporary directory; I think tests make a temp dir already for the workspace.